plugins/scaffolder-backend-module-github/src/actions/githubPullRequest.ts

```diff
@@ -419,6 +428,11 @@ export const createPublishGithubPullRequestAction = (
         }
         const response = await client.createPullRequest(createOptions);
 
+        if (createWhenEmpty === false && !response) {
+          ctx.logger.info('No changes to commit, pull request was not created');
+          return;
```

Comment:
Is it OK for this one to just ignore making any outputs and still report that it succeeded?

Reply:
The action is still considered successful in its execution. Setting the createWhenEmpty parameter to false basically translate to "create a PR if it is needed, pass through if not".